lib/logflare_web/router.ex

```diff
@@ -74,7 +74,7 @@ defmodule LogflareWeb.Router do
   end
 
   pipeline :require_mgmt_api_auth do
-    plug(LogflareWeb.Plugs.SetVerifyUser)
+    plug(LogflareWeb.Plugs.VerifyApiAccess)
```

Comment:
As I read it ... `VerifyApiAccess` will let ingest API keys in.

We should consider all `Logflare.User` `api_key`s exposed because they are used on in client side libs and potentially non-secure 3rd party forms (log drain forms).

Specific and obvious handling of an "ingest API key" vs a "management API key" is very important I think. 

I get the scent of RBAC here, which I like very much. But I'm not sure even with full RBAC we should ever let an API key have perms for ingest and management at the same time.

Reply:
underestand, definitely would need to implement scope checking for access tokens. Had done the PR with the idea that we would do a db migration to copy all existing `user.api_key` values over to the access tokens table. 

Can add in scope checking as well as separate PR.

RE Scope checking logic, we can have a `public` scope for non-secure use only (ingest only), `private` scope for everything, and also fine grained scope checking `ingest|endpoint_query|management`  if no scope is set (which is the case for current existing access tokens) then we consider it as "public", making this compatible with the above migration path as well.